H_ecto-models.md

```diff
@@ -517,6 +517,77 @@ end
 
 That's the end of our walk-through of Ecto usage in our controller actions. There is quite a bit more that Ecto models can do. Please take a look at the [Ecto documentation](http://hexdocs.pm/ecto/) for the rest of the story.
 
+### Data Relationship and Dependencies
+
+Handling individual tables is great, but if we want to build a modern web application, we will need a way to relate our data to each other. If we were Ruby on Rails refugees, we'll be glad to know Ecto provides a very familiar API for building relationships between models:
+
+`Schema.has_many/3` declares one to many relationships, for example, if we were building Youtube, our user model might have many uploaded video models.
```

Comment:
What would you think about using a more generic equivalent to Youtube? What about this?

. . . if we were building a video sharing service, our . . .